tests/test_humanize.py

```diff
@@ -47,6 +47,13 @@ def test_naturaltime(self, h):
         assert h._humanize(now) == 'now'
         assert h._humanize(now - one_month) == 'a month ago'
 
+    def test_naturaltime_with_utc(self, h, app):
+        app.config['HUMANIZE_USE_UTC'] = True
+        h.init_app(app)
+        assert  h._humanize(datetime.utcnow()) == 'now'
+        app.config['HUMANIZE_USE_UTC'] = False
+        h.init_app(app)
+
```

Comment:
This looks much better. I'm not very familiar with pytest. I have been using unittest. I will push the changes.